openmetadata-docs/content/partials/v1.6/deployment/upgrade/upgrade-prerequisites.md

```diff
@@ -113,75 +113,14 @@ We believe this update will bring greater consistency and clarity to our version
-
-#### Connector Schema Changes
-
-In the latest release, several updates and enhancements have been made to the JSON schema across various connectors. These changes aim to improve security, configurability, and expand integration capabilities. Here's a detailed breakdown of the updates:
-
-- **KafkaConnect**: Added `schemaRegistryTopicSuffixName` to enhance topic configuration flexibility for schema registries.
-- **GCS Datalake**: Introduced `bucketNames` field, allowing users to specify targeted storage buckets within the Google Cloud Storage environment.
-- **OpenLineage**: Added `saslConfig` to enhance security by enabling SASL (Simple Authentication and Security Layer) configuration.
-- **Salesforce**: Added sslConfig to strengthen the security layer for Salesforce connections by supporting SSL.
-- **DeltaLake**: Updated schema by moving metastoreConnection to a newly created `metastoreConfig.json` file. Additionally, introduced `configSource` to better define source configurations, with new support for `metastoreConfig.json` and `storageConfig.json`.
-- **Iceberg RestCatalog**: Removed clientId and `clientSecret` as mandatory fields, making the schema more flexible for different authentication methods.
-- **DBT Cloud Pipelines**: Added as a new connector to support cloud-native data transformation workflows using DBT.
-- **Looker**: Expanded support to include connections using GitLab integration, offering more flexible and secure version control.
-- **Tableau**: Enhanced support by adding capabilities for connecting with `TableauPublishedDatasource` and `TableauEmbeddedDatasource`, providing more granular control over data visualization and reporting.
-
-### Include DDL
-During the Database Metadata ingestion, we can optionally pick up the DDL for both tables and views. During the metadata ingestion, we use the view DDLs to generate the View Lineage.
-
-To reduce the processing time for out-of-the-box workflows, we are disabling the include DDL by default, whereas before, it was enabled, which potentially led to long-running workflows.
-
-### Secrets Manager
-Starting with the release 1.5.0, the JWT Token for the bots will be sent to the Secrets Manager if you configured one. It won't appear anymore in your dag_generated_configs in Airflow.
-
-### Python SDK
-The `metadata insight` command has been removed. Since Data Insights application was moved to be an internal system application instead of relying on external pipelines the SDK command to run the pipeline was removed.
+Now, we're changing this behavior to consider the success rate of all the steps involved in the workflow. The UI will
+then show more `Partial Success` statuses rather than `Failed`, properly reflecting the real state of the workflow.
```

Comment:
I dont understand how this impacts me as a user. Will I have more failing ingestions? Less? We can take an example ingestion and show how it would reflect differently now compared to before this change.

Reply:
approved, as a side quest I suggest we clarify definition of "partial success" or do away with it. From my experience ETL processes have a status of `success` / `failure` so the user knows if they need to inverine or not. I have suspicions that "partial success" can confuse platform users of whether they need to do something about their pipelines.

Examples of this design can be seen in:

- [Bash - nonzero for failure](https://tldp.org/LDP/abs/html/exit-status.html)
- [Airflow state machine](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#task-instances)
- [AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/functions-states.html)
- [Dagster](https://docs.dagster.io/_apidocs/execution#dagster.DagsterEvent.is_failure) - a bit less orthodox but a boolean value.